Parser/Models/Visit.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MongoDB.Bson;
+using System.Collections.Generic;
+
+namespace Visitors.Models
+{
+    public class Visit
+    {
+        public ObjectId Id { get; set; }
+        public int UniqueID { get; set; }
+        public int OwnerID { get; set; }
+        public BsonDateTime VisitDateTime { get; set; }
+        public int PageID { get; set; }
```

Comment:
There should be a whole object "Page" embedded into the Visit class (instead of keeping just ID of the page).